Classes/Views/PBGitRevisionCell.m

```diff
@@ -265,7 +246,7 @@ - (void) drawLabelAtIndex:(int)index inRect:(NSRect)rect
 	NSArray *refs = self.objectValue.refs;
 	PBGitRef *ref = [refs objectAtIndex:index];
 	
-	NSMutableDictionary* attributes = [self attributesForRefLabelSelected:[self isHighlighted]];
```

Comment:
I suspect this breaks the checked-out-branch-as-orange UI label.

Reply:
Thanks for the hint! The parameter of this method was never used, so I removed it. The regression was created due to a missing outlet connection to the history controller.